src/CsvReader.java

```diff
@@ -0,0 +1,50 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class CsvReader {
+
+    public static List<Map<String, String>> readCsv(String filename){
+        //Each List<String> will be a row of data
+        List<List<String>> rowsFromCsv = new ArrayList<>();
+        
+        try (BufferedReader incrementalClaimsData = (new BufferedReader(new FileReader(filename)))) {
+
+            //first row are headers. IGNORE THE FIRST ROW
+            incrementalClaimsData.readLine();
+
+            String dataRow;
+            while ((dataRow = incrementalClaimsData.readLine()) != null) {
+                String[] values = dataRow.split(", ");
+                rowsFromCsv.add(Arrays.asList(values));
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        
+        return mapValuesForEachRow(rowsFromCsv);
```

Comment:
We'd just return the list of rows here. This is safer than returning a list of maps as with maps you then have a bunch of operations like delete and put that can confuse other developers and let them do things that will break the code